Sage/carbon
# Improve accessibility of demo site tables

Add `caption` props to each table, and make the table headers render in a `thead`.

**NB** no release notes, as this is a demo site only fix.

# Related Issues / Pull Requests
#1157 

# Screenshots / Gifs
<img width="794" alt="rotor_tables" src="https://user-images.githubusercontent.com/52329/27233749-89f76184-52b2-11e7-93c6-90766a62a814.png">

# Testing Instructions
To test with VoiceOver on OS X: turn on with Cmd+F5, bring up the rotor using Ctrl+Option+U, use left / right arrows to find the Tables, and you should see the table listed via its caption.
